apps/web/components/eventtype/IntervalLimitsManager.tsx

```diff
@@ -0,0 +1,166 @@
+import { useAutoAnimate } from "@formkit/auto-animate/react";
```

Comment:
It is nice extracting this to its own components but it is not needed on multiple files. We can leave this on EventsLimitsTabs for now.